schemas/plugin_definitions/loaders.schema.json

```diff
@@ -0,0 +1,49 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "loaders.schema.json",
+    "title": "JSON Schema for loader plugins.",
+    "description": "https://docs.meltano.com/concepts/plugins#loaders",
+    "type": "object",
+    "allOf": [
+        {
+            "$ref": "common.schema.json"
+        },
+        {
+            "$ref": "loaders_specific.schema.json"
```

Comment:
I think if we want the YAML language server in IDEs to be able to pick these up and understand them, we need to specify the relative filepath

![image](https://user-images.githubusercontent.com/6589528/176008285-e126dabf-d8f0-44e3-877c-4e1c70cd1158.png)

Updating these to be `../common/common.schema.json` and `../common/loaders_specific.schema.json` will validate the plugin definitions inside a users IDE (if they have it set up to do so):

![image](https://user-images.githubusercontent.com/6589528/176008774-761f30c8-5825-420f-a6c3-07de845a26e5.png)

Might be valuable to improve the authoring experience (and maybe catch problems before CI even gets to it)